glitter/blocks/form/models.py

```diff
@@ -14,6 +15,13 @@ class BaseFormBlock(BaseBlock):
     class Meta:
         abstract = True
 
+    def get_email_field(self, form):
+        email = None
+        for name, field in form.base_fields.items():
+            if type(field) == EmailField:
+                email = form.cleaned_data[name]
```

Comment:
Maybe `break` here so it just returns the first field incase there are multiple?